thriftbp/errors.go

```diff
@@ -19,6 +19,16 @@ var (
 	_ BaseplateErrorCode = (*baseplatethrift.Error)(nil)
 )
 
+var (
```

Comment:
pro tip: golint also allows you to just have a single doc comment before the `var (` line for the whole var block, so you could just say something like `// ClientPoolConfig errors.` as the names are mostly self-explanatory.